Update issue templates (#5544)
filecoin-project/lotus: .github/ISSUE_TEMPLATE/mining-issues.md

---
name: Mining Issues
about: Create a report for help with mining failures.
title: "[Mining Issue]"
labels: hint/needs-triaging, area/mining
assignees: ''

---

Please provide all the information requested here to help us troubleshoot "mining/WinningPoSt failed" issues.

> If the information requested is missing, we will probably have to just ask you to provide it anyway,
> before we can help debug.

Reviewer: ```suggestion
If the information requested is missing, you may be asked you to provide it.
```
